scripts/deploy_assisted_service.sh

```diff
@@ -11,13 +11,25 @@ export SERVICE_PORT=$(( 6000 + $NAMESPACE_INDEX ))
 export SERVICE_BASE_URL=${SERVICE_BASE_URL:-"http://${SERVICE_URL}:${SERVICE_PORT}"}
 export EXTERNAL_PORT=${EXTERNAL_PORT:-y}
 export PROFILE=${PROFILE:-assisted-installer}
+export OCP_SERVICE_PORT=$(( 7000 + $NAMESPACE_INDEX ))
```

Comment:
not sure I understand, why should it run with another  port then the default assisted-service port 6000?
as I understand they are not overriding each other

Reply:
we should have only one per cluster, but 6000 is already used by the 'parent' service. so in order to reach both with same hostname, we use different ports.